fabric-convention-tags-v1/build.gradle

```diff
@@ -0,0 +1,44 @@
+archivesBaseName = "fabric-convention-tags-v1"
+version = getSubprojectVersion(project)
+
+moduleDependencies(project, [
+		'fabric-api-base'
+])
+
+sourceSets {
+	main {
+		resources {
+			srcDirs += [
+				'src/commontag/generated'
```

Comment:
Should probably exclude the `.cache` folder to save some file size. Also maybe change the folder name to `src/generated/resources`?

Reply:
Done.